File: Web/RMA-Web-Shell/projects/clientcare/src/app/product-manager/models/last-viewed-item.ts
import { ItemType } from './item-type.enum';

export class LastViewedItem {
    id: number;
    itemId: number;
    itemType: ItemType;
    user: string;
    date: Date;
    code = '';
    name = '';
    description = '';
    startDate: Date;
    endDate: Date;
    canEdit: boolean;
    modifiedBy: string;
    modifiedDate: Date;
    isActive: boolean;
    statusText: string;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Product/LastViewedItem.cs
using System;

namespace RMA.Service.ClientCare.Contracts.Entities.Product
{
    public class LastViewedItem
    {
        public int Id { get; set; }
        public int ItemId { get; set; }
        public string User { get; set; }
        public DateTime Date { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public bool CanEdit { get; set; }
        public string ModifiedBy { get; set; }
        public DateTime ModifiedDate { get; set; }
        public bool IsActive { get; set; }
        public string StatusText { get; set; }
    }
}